app/localization.js

```diff
@@ -48,11 +46,11 @@ Localizer.prototype = {
             }
 
             // First pass: perfect match
-            for (var j = 0;j < supportedLanguages.length;j++) {
-                var supLang = supportedLanguages[j];
-                supLang = supLang.toLowerCase();
-                supLang = supLang.replace("_", "-");
-                supLang = supLang.split("-");
+            for (let j = 0;j < supportedLanguages.length;j++) {
```

Comment:
I believe WebKit/Blink still has performance issues with `let` in loops, so we need to check that we don't get any performance regressions by using this.

Reply:
We can check that.
I assume that the performance difference is because using let make the loop into its own scope.

That's what you would expect if you come from any other language and makes things more solid in my opinion.

Reply:
Right, I really like these new features. But we need to make sure we aren't causing problems in other ways. A quick test using `vnc_playback.html` should answer that.